keyboards/montsinger/rebound/keymaps/rossman360/keymap.c

```diff
@@ -0,0 +1,55 @@
+#include QMK_KEYBOARD_H
+#include "rossman360.h"
+
+#define PGMOD LT(_NUM, KC_PGDN)
+#define TABMOD LT(_FN1, KC_TAB)
+#define SPCMOD LT(_FN1, KC_SPACE)
+#define ENTMOD LT(_FN2, KC_ENTER)
+#define ESCMOD LT(_NUM, KC_ESC)
+#define RSMOD LT(_FN1, KC_RSHIFT)
```

Comment:
IMO you should add these to `rossman360.h` but it's a matter of taste.

Reply:
well, I've got the same adv. keycode in another keymap that uses dif layer names, but I suppose I should sort that out

Reply:
Ah, makes sense.  You don't have to make that change.

In my case, rather than make a define for every layer tap, I do this:

```
#define ED(kc) LT(_EDITOR, KC_##kc)
#define NM(kc) LT(_NUMBER, KC_##kc)
#define SM(kc) LT(_SYMBOL, KC_##kc)
#define SC(kc) LT(_SECRET, KC_##kc)
#define FK(kc) LT(_F_KEYS, KC_##kc)
#define DB(kc) LT(_DEBUG,  KC_##kc)

```

Reply:
I should probably do that, personally. :)